linux-mister/config

```diff
@@ -3985,7 +4017,6 @@ CONFIG_DEBUG_BUGVERBOSE=y
 #
 CONFIG_DEBUG_INFO=y
 # CONFIG_DEBUG_INFO_REDUCED is not set
-# CONFIG_DEBUG_INFO_COMPRESSED is not set
```

Comment:
Try to not touch lines like these. It caused my PKGBUILD to start asking questions about DEBUG_INFO and KASAN.